client/me/account/main.jsx

```diff
@@ -84,25 +82,58 @@ const Account = React.createClass( {
 		debug( this.constructor.displayName + ' component is unmounting.' );
 	},
 
-	updateLanguage() {
-		let valueLink = this.valueLink( 'language' );
+	getUserSetting( settingName ) {
+		return this.props.userSettings.getSetting( settingName );
```

Comment:
Just pushed a commit where the `Account` component has `propTypes` declared and where all `userSettings` and `username` method calls are wrapped in `invoke`.